draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1191,9 +1191,9 @@ Inputs:
 
   chain:
   : An array of zero or more base64 encoded CA certificates. The first element
-    is the signer of the `submission`; the second certifies the first; etc.
+    is the certifier of the `submission`; the second certifies the first; etc.
     The last element of `chain` (or, if `chain` is an empty array, the
-    `submission`) either is, or is certified by, an accepted trust anchor.
+    `submission`) is certified by an accepted trust anchor.
```

Comment:
If the "last element of `chain` (or, if `chain` is an empty array, the `submission`)...is...an accepted trust anchor", and if that accepted trust anchor is not self-signed, then...well...that's what we're trying to disallow in this PR.

We do still allow inclusion of a trust anchor in the submission.  A self-signed trust anchor is certified by itself.